xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -106,12 +109,14 @@ impl<Network: Get<NetworkId>, AccountId: From<[u8; 32]> + Into<[u8; 32]> + Clone
 	Convert<MultiLocation, AccountId> for AccountId32Aliases<Network, AccountId>
 {
 	fn convert(location: MultiLocation) -> Result<AccountId, MultiLocation> {
-		let id = match location {
-			MultiLocation::X1(Junction::AccountId32 { id, network: NetworkId::Any }) => id,
-			MultiLocation::X1(Junction::AccountId32 { id, network })
-				if &network == &Network::get() =>
-				id,
-			l => return Err(l),
+		let id = match location.interior() {
```

Comment:
```suggestion
		let id = match (localtyion.parent_count(), location.interior()) {
```

All logic is correct to me, but maybe this is a nicer syntax